src/components/popover/popover/popover.stories.ts

```diff
@@ -114,7 +114,6 @@ const popover = (args: Args): TemplateResult => html`
       Some content.
       <sbb-link
         size="s"
-        variant="block"
```

Comment:
I'd vote to using block variant again. Because inline probably destabilizes chromatic builds again as the total width gets too large (just detected, I am sorry).

Reply:
no worries :)